docs/first-steps/querying/index.md

```diff
@@ -37,14 +37,38 @@ For a step-by-step guide on how to query data in Try Snowplow, see [this tutoria
 <Tabs groupId="warehouse" queryString>
   <TabItem value="postgres" label="Postgres" default>
 
-You can connect to the database using the username and password you provided when creating the pipeline, along with the `db_address` and `db_port` you noted down after the pipeline was created. (If you need to reset your username or password, you can follow [these steps](https://aws.amazon.com/premiumsupport/knowledge-center/reset-master-user-password-rds/).)
-
-Your database will contain two schemas:
+Your database will be named according to the `postgres_db_name` Terraform variable. It will contain two schemas:
 * `atomic` — the validated events
 * `atomic_bad` — the [failed events](/docs/understanding-your-pipeline/failed-events/index.md)
 
+You can connect to the database using the credentials you provided for the loader in the Terraform variables (`postgres_db_username` and `postgres_db_password`), along with the `postgres_db_address` and `postgres_db_port` Terraform outputs.
```

Comment:
@jbeemster When you review my changes, these are the ones to look out for — I hope I got the connection details correctly based on what the terraform changes looked like.